packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -147,20 +145,17 @@ interface SeparatorProps {
   topLevel: boolean
 }
 
-function Separator({ title, topLevel }: SeparatorProps) {
-  const hasTitle = title !== undefined
```

Comment:
Herr I'm using `!== undefined` check because title can be an empty string `''`

Reply:
@shuding fixed, added small comment also